src/Sylius/Component/Grid/Definition/Field.php

```diff
@@ -152,7 +152,11 @@ public function setEnabled($enabled)
      */
     public function setSortable($sortable)
     {
-        $this->sortable = $sortable ?: $this->name;
+        if ($sortable === true) {
+            $this->sortable = $this->name;
+        }
+
+        $this->sortable = $sortable;
```

Comment:
Don't we need any early return here? Otherwise `$this->sortable` will always be `$sortable`, no matter is it true or false.